sphinxemoji/utils.py

```diff
@@ -1,4 +1,5 @@
 import json
+from io import open
```

Comment:
I am guessing this is for Python 2 compatibility, but Python 2 is not supported by this project. Would you mind removing it? :blush: